src/screens/Home/HomeScreen.tsx

```diff
@@ -1,29 +1,75 @@
 import * as React from 'react';
-import { StyleSheet, View, TouchableOpacity, Text, Pressable} from 'react-native';
+import { StyleSheet, View, TouchableOpacity, Text, Pressable, Button, Modal, Image} from 'react-native';
 import EditScreenInfo from '../../components/EditScreenInfo';
 import { getAllTransactions } from '../../firebase/firestore/transaction';
 import { useEffect } from 'react';
 import ViewContainer from '../../components/ViewContainer';
 import { Divider, Title } from 'react-native-paper';
 import styles from './styles';
 import globalStyles from "../../globalStyles";
+import { ProgressSteps, ProgressStep } from 'react-native-progress-steps';
+import {moment} from 'moment';
+import StepIndicator from 'react-native-step-indicator';
+
+const labels = ["1000", '2000', '3000'];
+
+const getCurrentDate=()=>{
+    var moment = require('moment');
+    return (moment().format('dddd[,] MMMM Do')); 
+}
+
+const customStyles = {
+    stepIndicatorSize: 23.05,
+    currentStepIndicatorSize:23.05,
+    separatorStrokeWidth: 4.39,
+    currentStepStrokeWidth: 2,
+    stepStrokeCurrentColor: '#7F93D1',
+    stepStrokeWidth: 2,
+    stepStrokeFinishedColor: '#7F93D1',
+    stepStrokeUnFinishedColor: '#7F93D1',
+    separatorFinishedColor: '#7F93D1',
+    separatorUnFinishedColor: '#7F93D1',
+    stepIndicatorFinishedColor: '#253C85',
+    stepIndicatorUnFinishedColor: '#E6ECFE',
+    stepIndicatorCurrentColor: '#253C85',
+    stepIndicatorLabelFontSize: 14,
+    currentStepIndicatorLabelFontSize: 14,
+    stepIndicatorLabelCurrentColor: 'white',
+    stepIndicatorLabelFinishedColor: 'white',
+    stepIndicatorLabelUnFinishedColor: '#253C85',
+    labelColor: '#A9A9A9',
+    labelSize: 14,
+    currentStepLabelColor: '#253C85'
+  };
+
+
 
 const HomeScreen = ({navigation}: any) => {
     return (
         <View style = {styles.homeContainer}>
+            <View style = {styles.dateContainer}>
+                <Text style={[globalStyles.body2, {color: '#525454'}]}>{getCurrentDate()}</Text>
+            </View>
             <View style = {styles.headline}>
-                <Text style={[globalStyles.h3Bold, {color: '#525454'}]}>Kim Family</Text>
+                <Text style={[globalStyles.h3Bold, {color: '#272929'}]}>Kim Family</Text>
                 <View style={styles.innerContainer}>
-                    <View style={styles.addCircle} />
+                    <View style={styles.addPersonalCircle}>
+                    <Pressable
```

Comment:
Make sure to always watch your indents! the opening tag should also be at the same level as the closing tag!